packages/notices/src/store/actions.js

```diff
@@ -19,7 +19,7 @@ let uniqueId = 0;
 /**
  * Returns an action object used in signalling that a notice is to be created.
  *
- * @param {string}                [status='info']              Notice status.
+ * @param {string|undefined}      status                       Notice status ("info" if undefined is passed).
```

Comment:
The `status='info'` syntax makes "syntax" an optional argument in JSDoc. This actually isn't true -- status is still a _required_ parameter. It's just set to "info" if undefined is passed as the first argument.